test/unit/setup.js

```diff
@@ -22,6 +22,11 @@ function setUpDomEnvironment() {
   })
   const { window } = jsdom
 
+  // https://github.com/jsdom/jsdom/issues/1721
```

Comment:
This needs to come after `global.window` is redefined on line `30`. I'll do the change.